internal/server/routes.go

```diff
@@ -48,13 +48,22 @@ func (s *Server) GenerateRoutes(promRegistry prometheus.Registerer) *gin.Engine
 	api.GET("/.well-known/jwks.json", a.wellKnownJWKsHandler)
 
 	authn := api.Group("/", AuthenticationMiddleware(a))
-	get(a, authn, "/v1/identities", a.ListIdentities)
-	post(a, authn, "/v1/identities", a.CreateIdentity)
-	get(a, authn, "/v1/identities/:id", a.GetIdentity)
-	put(a, authn, "/v1/identities/:id", a.UpdateIdentity)
-	delete(a, authn, "/v1/identities/:id", a.DeleteIdentity)
-	get(a, authn, "/v1/identities/:id/groups", a.ListIdentityGroups)
-	get(a, authn, "/v1/identities/:id/grants", a.ListIdentityGrants)
+
+	get(a, authn, "/v1/users", a.ListUsers)
+	post(a, authn, "/v1/users", a.CreateUser)
+	get(a, authn, "/v1/users/:id", a.GetUser)
+	put(a, authn, "/v1/users/:id", a.UpdateUser)
+	delete(a, authn, "/v1/users/:id", a.DeleteUser)
+	get(a, authn, "/v1/users/:id/groups", a.ListUserGroups)
+	get(a, authn, "/v1/users/:id/grants", a.ListUserGrants)
+
+	get(a, authn, "/v1/identities", a.ListUsers)
+	post(a, authn, "/v1/identities", a.CreateUser)
+	get(a, authn, "/v1/identities/:id", a.GetUser)
+	put(a, authn, "/v1/identities/:id", a.UpdateUser)
+	delete(a, authn, "/v1/identities/:id", a.DeleteUser)
+	get(a, authn, "/v1/identities/:id/groups", a.ListUserGroups)
+	get(a, authn, "/v1/identities/:id/grants", a.ListUserGrants)
```

Comment:
I'm thinking we should do some kind of path redirect instead of this, which will duplicate the endpoints in the docs. Will discuss with you offline